src/cephadm/cephadm

```diff
@@ -119,6 +120,28 @@ class EndPoint:
         return f'{self.ip}:{self.port}'
 
 
+class ContainerInfo:
+    def __init__(self, container_id: str,
+                 image_name: str,
+                 image_id: str,
+                 start: str,
+                 version: str) -> None:
+        self.container_id = container_id
+        self.image_name = image_name
+        self.image_id = image_id
+        self.start = start
+        self.version = version
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, ContainerInfo):
+            return NotImplemented
+        return (self.container_id == other.container_id
+                and self.image_name == other.image_name
+                and self.image_id == other.image_id
+                and self.start == other.start
+                and self.version == other.version)
```

Comment:
what are we using this for? If it's just to see if two images we find are the same I feel like we only really need to check the image id (and there would be no reason to check things like container id and start time). I don't understand the use case for needing to check all these fields.

Reply:
this is used to compare two objects of type ContainerInfo. It's used in the UT of `get_container_info`: 

`                assert cd.get_container_info(ctx, daemon_filter, by_name) == output
`

Reply:
okay, if it's just being used by the UT and not to check if images are the same I guess it's fine.